frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -195,9 +204,23 @@ const LogsTabForPodName: React.FC<{ podName: string; isFailedPod: boolean }> = (
                       <ToolbarItem spacer={{ default: 'spacerSm' }} style={{ maxWidth: '200px' }}>
                         <SimpleDropdownSelect
                           isDisabled={podContainers.length <= 1}
-                          options={podContainers.map((container) => ({
-                            key: container.name,
-                            label: container.name,
+                          options={podStepStates.map((podStepState) => ({
+                            key: podStepState.stepName,
+                            label: podStepState.stepName,
+                            dropdownLabel: (
+                              <>
+                                {podStepState.stepName}{' '}
```

Comment:
I think to better match the design, we could get rid of this trailing space in the JSX, and instead wrap the name in a span and add a small margin using the `pf-v5-u-mr-sm` (patternfly utility) class name.